lib/organisation_import/import_school_data.rb

```diff
@@ -65,7 +65,7 @@ def school_is_local_authority_maintained?(row)
   end
 
   def school_is_community_or_foundation_special_school?(row)
-    row["EstablishmentTypeGroup (code)"] == "5" && %w[7 12].include?(row["TypeOfEstablishment (code)"])
+    row["EstablishmentTypeGroup (code)"] == "5" && %w[07 12].include?(row["TypeOfEstablishment (code)"])
```

Comment:
This is why I did the `.to_i` before if I recall correctly. The "5" is probably an "05" but ruby just goes with the flow

Reply:
I think it is consistent, it is always `5` or `05`. But yeah doing `to_i` is not too much work